atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -99,8 +99,10 @@ class KojiImportBase(ExitPlugin):
     key = None
     is_allowed_to_fail = False
 
+    args_from_user_params = map_to_user_params("userdata")
+
     def __init__(self, workflow, blocksize=None,
-                 target=None, poll_interval=5):
+                 target=None, poll_interval=5, userdata=None):
```

Comment:
Could the 'target' param be removed? I was confused about where it is coming from if not user params, but it turns out it is unused anyway

Reply:
sure will remove